src/test/java/com/google/sps/EventTest.java

```diff
@@ -0,0 +1,281 @@
+    {"f2", "v2"}, 
+    {"f1", "v1"}, 
+  }).collect(Collectors.toMap(data -> data[0], data -> data[1]));
+
+  @Test
+  public void testEvent_Event() {
+    Event event = Event.newBuilder()
+      .setOwnerID(USER_ID1)
+      .setGCalendarID(G_ID1)
+      .setTitle(TITLE1)
+      .setDescription(DESCRIPTION1)
+      .setCategory(CATEGORY1)
+      .setTags(TAGS1)
+      .setDuration(MINUTES_10)
+      .setLocation(LOCATION1)
+      .setLinks(LINKS1)
+      .setFields(FIELDS10)
+      .setInvitedIDs(Arrays.asList(USER_ID2))
+      .build();
+
+    Assert.assertNull(event.getID());
+    Assert.assertEquals(USER_ID1, event.getOwnerID());
+    Assert.assertEquals(G_ID1, event.getGCalendarID());
+    Assert.assertEquals(TITLE1, event.getTitle());
+    Assert.assertEquals(DESCRIPTION1, event.getDescription());
+    Assert.assertEquals(CATEGORY1, event.getCategory());
+    Assert.assertEquals(TAGS1, event.getTags());
+    Assert.assertFalse(event.isDateTimeSet());
+    Assert.assertNull(event.getDate());
+    Assert.assertNull(event.getTime());
+    Assert.assertNull(event.getDateTimeAsString());
+    Assert.assertEquals(MINUTES_10, event.getDuration());
+    Assert.assertEquals(LOCATION1, event.getLocation());
+    Assert.assertEquals(LINKS1, event.getLinks());
+    Assert.assertEquals(FIELDS10, event.getFields());
+    Assert.assertEquals(Arrays.asList(USER_ID2), event.getInvitedIDs());
+    Assert.assertEquals(new ArrayList<>(), event.getJoinedIDs());
+    Assert.assertEquals(new ArrayList<>(), event.getDeclinedIDs());
+  }
+
+  @Test
+  public void testEvent_equals_withEmptyEvents_returnsTrue() {
+    Event event1 = Event.newBuilder()
+      .setOwnerID(USER_ID1)
+      .build();
+
+    Event event2 = Event.newBuilder()
+      .setOwnerID(USER_ID1)
+      .build();
+
+    Assert.assertTrue(event1.equals(event2));
+    Assert.assertEquals(event1, event2);
+  }
+
+  @Test
+  public void testEvent_equals_withParameters_returnsTrue() {
+    Event event1 = Event.newBuilder()
+      .setOwnerID(USER_ID1)
+      .setGCalendarID(G_ID1)
+      .setTitle(TITLE1)
+      .setDescription(DESCRIPTION1)
+      .setCategory(CATEGORY1)
+      .setTags(TAGS1)
+      .setDuration(MINUTES_10)
+      .setLocation(LOCATION1)
+      .setLinks(LINKS1)
+      .setFields(FIELDS10)
+      .setInvitedIDs(Arrays.asList(USER_ID2))
+      .build();
+    
+    Event event2 = Event.newBuilder()
+      .setOwnerID(USER_ID1)
+      .setGCalendarID(G_ID1)
+      .setTitle(TITLE1)
+      .setDescription(DESCRIPTION1)
+      .setCategory(CATEGORY1)
+      .setTags(TAGS1)
+      .setDuration(MINUTES_10)
+      .setLocation(LOCATION1)
+      .setLinks(LINKS1)
+      .setFields(FIELDS11)
+      .setInvitedIDs(Arrays.asList(USER_ID2))
+      .build();
+
+    Assert.assertTrue(event1.equals(event2));
+    Assert.assertEquals(event1, event2);
+  }
+
+  @Test
+  public void testEvent_equals_withParameters_returnsNotEqual() {
```

Comment:
`withParameters` is not clear what you mean, especially it is the same as in the other test where equals returns true.
Maybe change it to `withDifferentEvents_returnsFalse`